src/circle2d.jl

```diff
@@ -1,9 +1,8 @@
 import LinearAlgebra: det
 
 #CirclePointRadius methods
-
-CirclePointRadius(x::Int, y::Int, ρ::T) where {T<:Real} = CirclePointRadius(Point(x,y), ρ)
-CirclePointRadius(p::CartesianIndex{2}, ρ::T) where {T<:Real} = CirclePointRadius(Point(p), ρ)
+CirclePointRadius(x::Int, y::Int, ρ::T,thickness::Int,fill::Bool) where {T<:Real} = CirclePointRadius(Point(x,y), ρ,thickness,fill)
```

Comment:
Need to avoid format issues like this.

```suggestion
CirclePointRadius(x::Int, y::Int, ρ::T, thickness::Int, fill::Bool) where {T<:Real} = CirclePointRadius(Point(x,y), ρ, thickness, fill)
```